src/main/java/org/folio/circulation/domain/Account.java

```diff
@@ -3,16 +3,35 @@
 import static org.folio.circulation.support.JsonPropertyFetcher.getNestedStringProperty;
 import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
 
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.Optional;
+
+import org.apache.commons.lang.math.NumberUtils;
+import org.joda.time.DateTime;
+
+import com.google.inject.internal.util.Lists;
+
 import io.vertx.core.json.JsonObject;
 
 public class Account {
 
   private final JsonObject representation;
+  private Collection<FeeFineAction> feeFineActions = Lists.newArrayList();
```

Comment:
Please don't use this dependency. Guice is not directly depended on in the POM file. It is indirectly by the `jacoco-maven-plugin` dependency. If we remove that dependency in the future, this code will not compile. Please use `java.util.ArrayList` directly.

Reply:
Done.